sequencing/vcf_query/Readme.md

```diff
@@ -1,21 +1,15 @@
 <!-- dx-header -->
 # vcf_query (DNAnexus Platform App)
 
-Calls "bcftools query" to extract annotations from the VCF file
+## What does this app do?
+Calls "bcftools query" to extract annotations from the VCF files (`*.vcf.gz`).
```

Comment:
This app calls `bcftools query` to extract annotations from VCF files (`*.vcf.gz`).